src/main/java/de/tum/in/www1/artemis/domain/enumeration/ExerciseType.java

```diff
@@ -12,4 +12,8 @@ public enum ExerciseType {
     public String getExerciseTypeAsReadableString() {
         return this.toString().toLowerCase().replace('_', ' ');
     }
+
+    public String getTypeAsStringForClient() {
```

Comment:
I somehow feel this method could have a better name 🤔 What exactly is this used for? What does it do? I feel like those questions are not answered by the method name. Also, some docstring would be nice :)

Reply:
Can you use `Exercise.getType()`? (It's not a method in `Exercise.java` yet, but all subclasses have it, I believe).